docs/system_overview/genquery.md

```diff
@@ -0,0 +1,443 @@
+#
+
+## iRODS General Query (GenQuery)
+
+GenQuery is the SQL-like syntax used to query the iRODS catalog. It is used throughout the server to accomplish everything required of the system from resource hierarchy resolution to enforcing permissions, as well as directly by users via `iquest` and other clients.
+
+This page describes syntax for GenQuery as it has existed in iRODS for almost as long as it has existed. A more modern solution is now available in the iRODS server called GenQuery2 which will eventually replace the legacy GenQuery implementation.
```

Comment:
The part about how long GenQuery has existed reads a little weird to me. Consider rephrasing that part.

Reply:
I agree. Must have been tired when I wrote that lol

Reply:
I reworded the phrase. I'll resolve this but feel free to suggest more tweaks